config/locales/es.yml

```diff
@@ -37,6 +37,7 @@ es:
         missing_file:                     "No se pudo descargar %{file_name}. Puede que el archivo no haya
                                            sido encontrado."
         empty_file_warning: "UPDATE ME"
+      account_settings: 'UPDATE ME'
```

Comment:
Don't add this here; add it under `views/users/es.yml` (and similar for the other locales). But I suppose you can keep the typo fix.